tests/copp/test_copp.py

```diff
@@ -247,6 +247,30 @@ def _teardown_testbed(dut, creds, ptf, test_params):
         logging.info("Reloading config and restarting swss...")
         config_reload(dut)
 
-    logging.info("Restore LLDP")
-    dut.command("docker exec lldp supervisorctl start lldpd")
-    dut.command("docker exec lldp supervisorctl start lldp-syncd")
+
+@pytest.fixture(scope="class")
+def disable_lldp_for_testing(
```

Comment:
Can this logic not be included in `_setup_testbed` and `_teardown_testbed`? If I read it correctly, `_teardown_testbed` will always be called even if there is a failure as it is now inside `finally` block.

Reply:
There are other steps in `teardown` that could fail which would cause it to still be missed. I think the rest of the steps should be fixture-ized as much as possible to avoid this, but 1) I can't test this at the moment (because the COPP test itself has issues w/ the recent COPP changes) and 2) it would make this PR a lot more complex.